src/stateProviders/provideSavedQueries.js

```diff
@@ -42,6 +45,7 @@ export default provideState({
         .then(r => state => ({
           ...state,
           queries: state.queries.filter(({ id }) => id !== queryId),
+          exampleQueries: state.exampleQueries.filter(({ id }) => id !== queryId),
```

Comment:
I pushed this quick fix for now, but much prefer we did not have these extra split between `queries` and `exampleQueries` in the state and just do that splitting at render. The visual difference is really more a concern of the consuming component rather than the state. 

Would love to just change this guy to a render prop one day too...